### DISCUSSION TITLE
Manually validate a ticket registration

### DISCUSSION THREAD

**Discussion by olivierlambert** [pretix/pretix#4221] [Q&A]
Hello everyone!

First, thanks for this amazing software. I started to explore it, and it seems very powerful.

I'm about to organize a event that will be free, but free doesn't mean no control on who will attend and the max number of attendees.

That's why I wonder if there's a way to configure an event with free ticket BUT leaving an admin to validate the registration, in order to be sure we have only people that should be there (like related to the event, not fully unknown people or even people just registering for the sake of taking seats).

Is it possible?

**Comment by raphaelm** [comment]
Yes, you need to check the checkbox on the product that is named something like "require approval"

**Comment by olivierlambert** [comment]
Found it!

![image](https://github.com/pretix/pretix/assets/1241401/c7e69bc4-fb4e-49e2-89f2-c7e4fd3f9435)

Thanks! I will test that to see if it's exactly what we need, but it seems to be. Thank you very much for your quick answer :+1:
